nemo/utils/exp_manager.py

```diff
@@ -658,24 +657,6 @@ def get_git_diff():
         return "{}\n".format(err.output.decode("utf-8"))
 
 
-class LoggerList(_LoggerCollection):
```

Comment:
This is not available in Lightning any more (it was moved to the `_grave_yard` which is pretty aggressive). I've removed it for now, but I need to check what the disadvantage of this is.

Reply:
That's painful, though I wonder what we needed Nemo name and version for

Reply:
From my look around, they were not being used